connector/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaSourceProviderSuite.scala

```diff
@@ -62,6 +140,7 @@ class KafkaSourceProviderSuite extends SparkFunSuite {
         stream => {
       assert(expected.toLong === stream.kafkaOffsetReader.offsetFetchAttemptIntervalMs)
     })
+    SparkEnv.set(null)
```

Comment:
Is this to fix some test failure ?

Reply:
Yeah this was initially part of the `beforeAll` command on line 35 but I removed it since the new tests rely on the spark env config set by the `SharedSparkSession`, I moved it here since the spark config gets modified by the methods in this test. Alternatively, we could set it back to what it was at the start which seems like a safer option.